UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/AuthTokenProvider.cs

```diff
@@ -0,0 +1,126 @@
+﻿using JWT;
+using JWT.Algorithms;
+using JWT.Serializers;
+using Microsoft.Identity.Client;
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using static UKHO.ExchangeSetService.API.FunctionalTests.Helper.TestConfiguration;
+
+namespace UKHO.ExchangeSetService.API.FunctionalTests.Helper
+{
+    public class AuthTokenProvider
+    {
+        static string EssAccessToken = null;
+        static string EssAccessTokenNoAuth = null;
+        static EssAuthorizationTokenConfiguration EssauthConfig = new TestConfiguration().EssAuthorizationConfig;
```

Comment:
Could make this readonly

Reply:
We cannot make it readonly since we are  assigning the value in the public method. static readonly cannot be assigned to (except in a static constructor or a variable initializer)

Reply:
Sure, thanks.